### ISSUE TITLE
Raspberry Pi2/3 support officially 

### ISSUE THREAD

**Issue by sneakernets** [OpenArena/engine#22]
The "Raspberry Pi version" of OA has mysteriously disappeared, the ioquake3 Raspberry Pi build is over 4 years old, and the latest ioquake3 requires *compiling your own kernel* to build properly on Pi.

The OpenGL wrapper for the Pi 3 is experimental, and while it "works", it's extremely slow (worse than Rage Pro performance easily), and crash-prone. So that's not a good solution either.

Perhaps an no-nonsense Pi port is in order? Tons of deb packages of everyone reinventing the wheel, but no source code.

**Comment by leilei-** [comment]
I don't have a Pi. maybe you can do it ;)

Just remember that we're using renderer_oa, because that's where allt he neat things are

**Comment by Hiradur** [comment]
OpenArena installed from the Debian repositories worked quite well for me with the experimental OpenGL driver enabled on a RPi2 a few months ago and it's probably only gotten better now.

Here is my tweaked q3config for RPi2: https://gist.github.com/Hiradur/9c396b54ea4213d0de62

**Comment by leilei-** [comment]
All I can think of is some GLES 2.0 Pi renderer and/or maybe a speed-focused software renderer for those who don't enable the GL driver.

**Comment by symbios24** [comment]
Openarena is not working anymore with raspbian stretch by following the known method under:

sudo apt-get install openarena

wget http://www.berryterminal.com/dl/ioquake3_99.1.36-rpi01_armhf.deb

sudo dpkg -i ./ioquake3_99.1.36-rpi01_armhf.deb

its not running at all is there a fix for this?

**Comment by symbios24** [comment]
Openarena is not working anymore with raspbian stretch by following the known method under:

sudo apt-get install openarena

wget http://www.berryterminal.com/dl/ioquake3_99.1.36-rpi01_armhf.deb

sudo dpkg -i ./ioquake3_99.1.36-rpi01_armhf.deb

its not running at all is there a fix for this?

**Comment by leilei-** [comment]
Yeah, the fix is using the correct repository tha we do support (i.e. this one you're making an issue in) and building from it

**Comment by symbios24** [comment]
great fix it help me a lot to do nothing

**Comment by WiccyShackleton** [comment]
I found that rebuilding the kernel fixed a lot of graphics issues (low FPS etc) on the raspberry pi 4,  havent tried on the 2 or 3 though.
